packages/client/hmi-client/src/components/widgets/tera-infinite-canvas.vue

```diff
@@ -106,7 +119,7 @@ const resizeObserver = new ResizeObserver(() => updateDimensions());
 
 onMounted(() => {
 	const svg = d3.select(backgroundLayerRef.value as SVGGElement); // Parent SVG
-	const container = svg.append('g'); // Pan/zoom area
+	const edges = d3.select(edgesRef.value as SVGGElement); // Pan/zoom area
```

Comment:
There are a few things here not quite right - namely the slot system is being bypassed in a strange manner. Let's set up a bit of time to clear up our understandings, maybe we can setup a pair-programming session kind of thing.